qir/qsharp/TargetPackages/QSharpVersion/Example.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.Quantum.Sdk/0.24.41925-alpha">
```

Comment:
The two project files can now be edited to look like similar to this:
`<Project Sdk="Microsoft.Quantum.Sdk/0.24.41925-alpha">

  <PropertyGroup>
    <QscVerbosity>Detailed</QscVerbosity>
    <OutputType>Exe</OutputType>
    <TargetFramework>net6.0</TargetFramework>
    <ExecutionTarget>quantinuum.qpu</ExecutionTarget>
    <RuntimeCapability>AdaptiveExecution</RuntimeCapability>
  </PropertyGroup>

</Project>
`
The RuntimeCapability is only needed for the Quantinuum target since it supports multiple formats. For the other ones you can/should remove it. As mentioned, project properties can also be set as part of the dotnet build command.